Source/Renderer/UniformState.js

```diff
@@ -885,10 +906,7 @@ define([
         this._resolutionScale = canvas.width / canvas.clientWidth;
 
         var camera = frameState.camera;
-
-        setView(this, camera.viewMatrix);
-        setInverseView(this, camera.inverseViewMatrix);
-        setCamera(this, camera);
+        this.updateCamera(camera);
```

Comment:
Add a TODO to your roadmap since those tests should probably call `update` and `updateCamera` (or we can make a tests-specific helper that calls both).  I hate to make the render workflow more complicated so we don't have to change tests.